website/content/en/preview/getting-started/getting-started-with-karpenter/karpenter-performance-dashboard.json

```diff
@@ -1,573 +1,595 @@
 {
     "annotations": {
```

Comment:
Can you also make these change in v1.0 and docs folders?

Reply:
@engedaam Good point, thx. Done.